Makefile

```diff
@@ -29,5 +29,5 @@ upgrade:
 	helm upgrade connaisseur helm  -n $(NAMESPACE) --wait
 
 annihilate:
-	kubectl delete all,mutatingwebhookconfigurations,clusterroles,clusterrolebindings,configmaps,imagepolicies,secrets,serviceaccounts,crds -lapp.kubernetes.io/instance=connaisseur
+	kubectl delete all,mutatingwebhookconfigurations,clusterroles,clusterrolebindings,configmaps,secrets,serviceaccounts,crds -lapp.kubernetes.io/instance=connaisseur
```

Comment:
this change was already made in #255 and I think we should not do this for some time to ensure smooth upgrading to the latest version. It should not fail in case there is no imagepolicy, right?

Reply:
it does fail :crying_cat_face:

Reply:
as discussed...might be fixed by deleting separately with silent failing

Reply:
and it is still a duplicate and will cause a rebase issue